### PR TITLE
Web summit application

### PR DESCRIPTION
**Pull request by mirwaisfarahi**
This web application has been created with home and about pages which can be used as an application for events and conferences. The following features has been created with this application:

1. The home page.
2. The about page.
3. Links
-   The home page have a link in the menu to the about page.
-   The logo in the header links to the home page.

4. Mobile menu
5. Dynamic page: The section "Featured speakers"  created dynamically in JavaScript.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,13 +1,139 @@
...
+                </div>
+
+                <div class="program">
+                    <img src="images/stage.png" alt="Lecture">
+                    <h4>WS Exhibition</h4>
+                    <p>Appreciate various creations inluding new web Technologies.</p>
+                </div>
+
+                <div class="program">
+                    <img src="images/forum.png" alt="Lecture">
```
Comment (AnselemOdims) [inline index.html:81]: - [x] Please ensure that the program section cards are all same size as  it is in the original design.

#### Your Design
image

### Original Design
image

File: about.html [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                    <img src="images/riot-logo.png" alt="Riot Logo">
+                </div>
+                <div class="sponsor">
+                    <img src="images/sony-logo.png" alt="Play Station">
+                </div>
+                <div class="sponsor">
+                    <img src="images/xbox-logo.png" alt="Play Station">
+                </div>
+                <div class="sponsor">
+                    <img src="images/nintendo-logo.png" alt="Play Station">
```
Comment (AnselemOdims) [inline about.html]: - [x] Please kindly remove the sponsor's section as it is not specified in the 
original design

### Your Design
image

### Original Design
image

**Review by AnselemOdims — CHANGES_REQUESTED** [review]
### Hi @mirwaisfarahi, 
You have been doing a tremendous job so far :ok_hand:
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
No linter errors :heavy_check_mark:

## Required Changes ♻️
- [x] Please ensure that the presentation video does not exceed 5mins
- [x] Please also talk about one thing/code you are really proud of about this project.
- [x] Please also check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity! You can tag me using @AnselemOdims for a quicker response

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by codepantha — APPROVED** [review]
Hi @Mirwais-Farahi 👋,

## STATUS: APPROVED ✔️  

Well done implementing the required changes. You have a really beautiful project and you have closely followed the design guidelines.
This project is now complete!  Congratulations! 🎉 



### Highlights ⭐ 
✔️ Descriptive PR
✔️ Beautiful design
✔️ Linter checks

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**